nle/env/base.py

```diff
@@ -511,6 +532,38 @@ def get_seeds(self):
         """
         return self.env.get_current_seeds()
 
+    def tty_render(self, chars, colors, print_guides=False):
+        # tty_chars_index = self._observation_keys.index(keys[0])
+        # tty_colors_index = self._observation_keys.index(keys[1])
+        # tty_chars = obs[tty_chars_index]
+        # tty_colors = obs[tty_colors_index]
+
+        tty_chars = chars
+        tty_colors = colors
+
+        H, W = tty_chars.shape
+        rendering = []
+        col_index_str = " " + COLORS[1] + "".join([str(i % 10) for i in range(W)])
+        if print_guides:
+            rendering.append(col_index_str)
+        for i in range(H):
+            line = []
+            for j in range(W):
+                char = tty_chars[i, j]
+                color = tty_colors[i, j]
+                line.append(COLORS[color] + chr(char))
+            row_index_str = COLORS[1] + str(i % 10)
+            if print_guides:
+                rendering.append(row_index_str + "".join(line) + row_index_str)
+            else:
+                rendering.append("".join(line))
+        if print_guides:
+            rendering.append(col_index_str)
+        rendering.append(COLORS[7])
+        rendering = "\n".join(rendering)
+        print(rendering)
```

Comment:
```suggestion
```

Reply:
I suggest moving the printing to a calling function so we can reuse this function for say, logging,